_pkgdown.yml

```diff
@@ -23,6 +23,14 @@ reference:
   desc: "Functions to manage projects."
   contents:
   - has_concept("project-management")
+- title: Datasets
```

Comment:
So cool to see all this implemented!

Question for you: did you consider using `Entity Lists` here? We use that term everywhere user-facing, now, including form design. The only place `Datasets` shows up is in the API. `ruODK` kind of sits between the lands of users and developers. I think I slightly lean towards "entity lists" but not very strongly. It depends on whether you think ruODK users go directly to the API docs much or not.

I'd like `pyodk` to align with you on that decision.

Reply:
Thanks for the context here!

They initially used term Datasets is taken from the API docs but I want to use what makes most sense.

I feel Entity Lists makes most sense here, because it's the term you use user facing and it also explains its relationship with Entities.

Reply:
If you're liking "entity list" for headings, it feels like function names should match. `entity_list_list` is a little unfortunate but there's precedent in e.g. the [Netflix LOLOMO](https://twitter.com/arungupta/status/624402051116568576).

The strongest argument I can think of in favor of "entity list" is that it's likely that a lot of the users of the wrappers don't go to the source API docs. In that case, they'd be exposed to "entity list" exclusively so using that in the wrappers would keep their world more coherent. The power users who go to the source API docs are the least likely to be confused by the discrepancy. 

@lindsay-stevens, looping you in for a `pyodk` perspective. Do you have a preference between the API-only "dataset" term or the user-facing "entity list" for these API wrapper libraries?

Reply:
No preference between the terms as such, so long as we stick to one of them as consistently as possible

Reply:
To reduce confusion about the duplicate `list` but go with @lognaturel's suggestion, how do you feel about `entitylist_list` and `entitylist_detail`?

Reply:
That seems like it's mostly up to ruODK conventions but it does sound reasonable to me! I think `pyodk` will likely end up with something like `client.entity_lists.list()`.

Reply:
Sounds great! 

I'm open to suggestions and happy to go with anything that makes more sense. For now `ruODK` will go with `entitylist_{list, detail}` and get rid of the term `dataset` in functions altogether. Then the only difference to `pyODK`'s `client.entity_lists.list()` are the respective naming conventions.